File: src/main/java/com/vjiki/music/service/PlaylistService.java
package com.vjiki.music.service;

import java.util.List;
import java.util.UUID;

import com.vjiki.music.dto.PlaylistResponse;
import com.vjiki.music.dto.PlaylistWithSongsResponse;

public interface PlaylistService {

    List<PlaylistResponse> getPlaylistsByUserId(UUID userId);

    PlaylistWithSongsResponse getPlaylistWithSongs(UUID playlistId);
}

File: src/main/kotlin/com/vjiki/music/service/PlaylistService.kt
package com.vjiki.music.service

import com.vjiki.music.dto.PlaylistResponse
import com.vjiki.music.dto.PlaylistWithSongsResponse
import java.util.UUID

interface PlaylistService {
    fun getPlaylistsByUserId(userId: UUID): List<PlaylistResponse>
    fun getPlaylistWithSongs(playlistId: UUID): PlaylistWithSongsResponse
}

